infra.bs

```diff
@@ -119,6 +119,29 @@ will make the caller automatically rethrow the given value, if any, and thereby
 caller's algorithm. Using prose the caller has the ability to "catch" the exception and perform
 another action.
 
+<h4 id=algorithm-iteration>Iteration</h4>
+
+<p>There's a variety of ways to repeat a set of steps until a condition is reached. The Infra
+Standard is not (yet) exhaustive on this, but here are some examples:
+
+<ul>
+ <li><p>For each, as demonstrated later in this section.
+
+ <li>
+  <p>While
```

Comment:
I think maybe the value is in telling people to use "while" instead of "repeat" or "perform the following steps repeatedly" or similar. Infra has a lot of terms that can feel silly to link to, and this falls pretty far on the silly end of the spectrum.